payment/payment.go

```diff
@@ -30,6 +30,13 @@ func FromPublicKey(pubkey *btcec.PublicKey, network *network.Network) Payment {
 	return Payment{network, pubkey, hash, nil, nil, nil}
 }
 
+// FromPayment creates a Payment struct from a another Payment
+func FromPayment(payment *Payment, network *network.Network) Payment {
+	buf := payment.Hash
+	hash := hash160(buf)
```

Comment:
Here we should hash the payment `Script`, instead of the `Hash`

Reply:
It was done based on comment:
https://github.com/vulpemventures/go-elements/issues/9#issuecomment-628507338
"This technically means taking the hash of the given payment and hashing it again to produce the final hash that will be encoded"
@altafan do you think a comment should be updated form 'taking the hash' to 'taking the script'?
Please review if PR is ok now.

Reply:
Yes, thanks for pointing this out. Already updated the comment.

Reply:
@altafan When the Script file it's valorized? The FromPublicKey, FromPubKeys etc... should do that then?